liveblog.php

```diff
@@ -1169,6 +1169,10 @@ public static function add_liveblog_to_content( $content ) {
 			$liveblog_output = '<div id="wpcom-liveblog-container" class="' . self::$post_id . '"></div>';
 
 			$liveblog_output = apply_filters( 'liveblog_add_to_content', $liveblog_output, $content, self::$post_id );
+			
+			if ( false === apply_filters( 'liveblog_output_at_bottom', true ) ) {
```

Comment:
I'm not sure why but to me it feels more obvious to have the filter that changes something to be "liveblog_output_at_top" and have it default to false. I'm having a hard time to vocalize it but it seems like a negation that doesn't need to be there. Like the question is liveblog_output_at_bottom and then having it to true will will then output it to the top. It works, but it "feels" strange to me.

Reply:
I think that makes sense; it kind of feels like a double negative when folks want to actually have output at the top, whereas a default false for 'liveblog_output_at_top' feels more straightforward.